cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -395,6 +401,18 @@ func CreateControllerContext(s *config.CompletedConfig, rootClientBuilder, clien
 	versionedClient := rootClientBuilder.ClientOrDie("shared-informers")
 	sharedInformers := informers.NewSharedInformerFactory(versionedClient, ResyncPeriod(s)())
 
+	// Resource quota controller needs to see all Uninitialized resources.
```

Comment:
Aren't we removing this concept? Do we need to implement this? cc @smarterclayton

Reply:
I think at this point if we can reach agreement on namespace initialization (and its needs) not using the generic mechanism or fields, then we probably will remove the generic mechanism and fields.  Let's put a hold on this until we can have that discussion, maybe next api machinery meeting.

Reply:
Agree.